pages/app/styles.scss

```diff
@@ -14,11 +14,12 @@ body {
 /* stylelint-disable-next-line selector-class-pattern */
 :global(.awsui-motion-disabled) {
   // disable blinking caret for stable screenshot tests
-  // stylelint-disable-next-line @cloudscape-design/no-implicit-descendant, selector-max-type
+  // stylelint-disable @cloudscape-design/no-implicit-descendant, selector-max-type
```

Comment:
Stylelint changed its behavior. `disable-next-line` literally means "next line", for multiline selectors we have to use disable/enable combination